src/components/AppModal.vue

```diff
@@ -44,6 +44,8 @@ export default {};
   position: relative;
   width: 80vw;
   max-width: 700px;
+  max-height: 100vh;
```

Comment:
Might actually look better at 90vh or thereabouts than if it takes up the full screen

Reply:
True!, it's better with 90vh, PR Updated 👍